L/LibCURL/build_tarballs.jl

```diff
@@ -3,12 +3,14 @@
 using BinaryBuilder
 
 name = "LibCURL"
-version = v"7.68.0"
+version = v"7.70.0"
 
 # Collection of sources required to build LibCURL
 sources = [
-    "https://curl.haxx.se/download/curl-$(version).tar.bz2" =>
-    "207f54917dd6a2dc733065ccf18d61bb5bebeaceb5df49cd9445483e8623eeb9",
+    ArchiveSource("https://curl.haxx.se/download/curl-$(version).tar.gz", 
+    "ca2feeb8ef13368ce5d5e5849a5fd5e2dd4755fecf7d8f0cc94000a4206fb8e7"), 
+    FileSource("https://curl.haxx.se/ca/cacert.pem", 
```

Comment:
```suggestion
    FileSource("https://curl.haxx.se/ca/cacert-2020-01-01.pem", 
```

Let's use the versioned filename, so that we don't have this script fail when we try to re-run it in the future (or even worse, we think we're updating and getting the latest, but we're just pulling an old version out of the BB download cache)

Reply:
The versioned files are historical, the current file doesnt have a version on it. So while I understand the downsides, I am loath to not use the latest file.

Reply:
Anyway when we want to build a new version you need to update the hash

Reply:
Actually, no, that is not true, even though they use the heading "Older"